.travis.yml

```diff
@@ -14,7 +14,6 @@ script:
 
 after_script:
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
-  - NODE_ENV=test istanbul cover ./node_modules/mocha/bin/_mocha --report lcovonly -- -R spec && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
 
 
 after_success:  npm run coveralls
```

Comment:
Try removing this line
Your travis may pass afterwards

Reply:
Okay